app/src/main/java/com/example/bigowlapp/activity/SignUpPageActivity.java

```diff
@@ -40,12 +41,17 @@ protected void initialize() {
             String email = userEmail.getText().toString();
             String pass = userPassword.getText().toString();
             String userPhone = this.userPhone.getText().toString();
-            String userName = username.getText().toString();
+            String firstName = userFirstName.getText().toString();
+            String lastName = userLastName.getText().toString();
+
 
             //Error handling
-            if (userName.isEmpty()) {
-                username.setError("Please enter your name");
-                username.requestFocus();
+            if (firstName.isEmpty()) {
+                userFirstName.setError("Please enter your first name");
+                userFirstName.requestFocus();
+            } else if (lastName.isEmpty()) {
+                userLastName.setError("Please enter your first name");
```

Comment:
Lol. Copy paste error. You mean:
 `userLastName.setError("Please enter your last name");`
wrong string

Reply:
thats true....

Reply:
I changed the string